client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx

```diff
@@ -81,213 +98,198 @@ beforeEach(() => {
-      generateApiMock(
-        sampleSizeMock,
-        contestMocks.filledTargetedWithJurisdictionId,
-        {
-          jurisdictions: [],
-        }
-      )
-    )
     auditSettingsMock.mockReturnValue(settingsMock.full)
-    const { container } = render(
-      <Router>
-        <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
-      </Router>
-    )
-    await waitFor(() => expect(apiMock).toHaveBeenCalled())
-    expect(container).toMatchSnapshot()
+    const expectedCalls = [
+      apiCalls.getEmptyJurisdictions,
+      apiCalls.getJurisdictionFile,
+      apiCalls.getTargetedContests,
+      apiCalls.getSampleSizeOptions,
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      const { container } = renderView()
+      await screen.findByText('Review & Launch')
+      expect(container).toMatchSnapshot()
+    })
   })
 
   it('renders despite missing jurisdictions on opportunistic contest', async () => {
-    apiMock.mockImplementation(
-      generateApiMock(
-        sampleSizeMock,
-        contestMocks.filledOpportunisticWithJurisdictionId,
-        {
-          jurisdictions: [],
-        }
-      )
-    )
     auditSettingsMock.mockReturnValue(settingsMock.full)
-    const { container } = render(
-      <Router>
-        <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
-      </Router>
-    )
-    await waitFor(() => expect(apiMock).toHaveBeenCalled())
-    expect(container).toMatchSnapshot()
+    const expectedCalls = [
+      apiCalls.getEmptyJurisdictions,
+      apiCalls.getJurisdictionFile,
+      apiCalls.getOpportunisticContests,
+      apiCalls.getSampleSizeOptions,
+    ]
+    await withMockFetch(expectedCalls, async () => {
+      const { container } = renderView()
+      await screen.findByText('Review & Launch')
+      expect(container).toMatchSnapshot()
+    })
   })
 
   it('launches the first round', async () => {
-    apiMock.mockImplementation(
-      generateApiMock(
-        sampleSizeMock,
-        contestMocks.filledTargetedWithJurisdictionId,
-        { jurisdictions: jurisdictionMocks.allManifests }
-      )
-    )
-    render(
-      <Router>
-        <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
-      </Router>
-    )
-    const launchButton = await screen.findByText('Launch Audit')
-    fireEvent.click(launchButton, { bubbles: true })
-    await screen.findByText('Are you sure you want to launch the audit?')
-    const confirmLaunchButton = screen.getAllByText('Launch Audit')[1]
-    fireEvent.click(confirmLaunchButton, { bubbles: true })
-    await waitFor(() => {
-      expect(apiMock).toHaveBeenCalledTimes(5)
-      expect(apiMock.mock.calls[4][1]).toMatchObject({
-        body: JSON.stringify({
-          sampleSize: 46,
-          roundNum: 1,
-        }),
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        method: 'POST',
-      })
+    auditSettingsMock.mockReturnValue(settingsMock.full)
```

Comment:
any reason this is mocked while all the other data sources use the mock API?

Reply:
@jonahkagan I think that's a holdover from the old tests, before mocking the api was simplified. The settings were tested directly, and adding those mocks into the mix would have made a horrible mess. We could integrate the settings into the sample size api mocks now, I think, though this is still cleaner imo.